controllers/component_build_controller_pac.go

```diff
@@ -292,7 +292,8 @@ func (r *ComponentBuildReconciler) TriggerPaCBuild(ctx context.Context, componen
 	triggerURL := fmt.Sprintf("%s/incoming?secret=%s&repository=%s&branch=%s&pipelinerun=%s", webhookTargetUrl, secretValue, repository.Name, targetBranch, pipelineRunName)
 	HttpClient := GetHttpClientFunction()
 
-	resp, err := HttpClient.Post(triggerURL, "application/json", nil)
+	formParams := url.Values{"source_url": []string{repoUrl}}
```

Comment:
Would be nice to have explaining comments here and in other places

Reply:
adding